src/services/prisma-datastore/schema.prisma

```diff
@@ -0,0 +1,92 @@
+
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["extendedIndexes"]
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Project {
+  id            Int       @id @default(autoincrement())
+  createdAt     DateTime  @default(now()) @map("created_at")
```

Comment:
Not a fan of tracking things that aren't in service of a user story. What's the justification for `createdAd`?